### PR TITLE
Merging jedsinger's scattered edits into Master

### PR DESCRIPTION
**Pull request by jedsinger**

### CODE REVIEW

File: code/@ColorModel/ColorModel.m [hunk 1]
```diff
@@ -42,7 +42,7 @@
         CM.color_translation_model=[] ;% For converting other channels to ERF channel AU equiv
         CM.noise_model=[]             ;% For understanding the expected constitutive expression noise
         CM.filters={};                 % filters to remove problematic data (e.g. debris, time-contamination)
-        CM.standardUnits = 'not yet set';  % Should be the value from column E in BeadCatalog.xlsx
+        CM.standardUnits = 'Fluorescence';  % Should instead be the value from column E in BeadCatalog.xlsx
```
Comment (jakebeal) [inline code/@ColorModel/ColorModel.m]: What is the purpose of this change? We've previously left this set to 'not yet set' until resolution, which (should) set it to either the right ERF or "arbitrary units" (thought there is an issue outstanding).

File: code/@ColorModel/beads_to_ERF_model.m [hunk 2]
```diff
@@ -59,6 +59,9 @@
 TASBESession.succeed('TASBE:Beads','ObtainBeadPeaks','Found specified bead model and lot');
 
 
+TASBESession.succeed('TASBE:Beads','ObtainBeadPeaks','Found specified bead model and lot');
```
Comment (jakebeal) [inline code/@ColorModel/beads_to_ERF_model.m]: Please remove this duplicate.

**Comment by jakebeal** [comment]
Once I resolved the merge conflicts, it looks like there are almost no issues left. The one thing I saw was the point cloud CSVs being optional, which I believe we've dealt with at a higher level, so I removed that piece.  If that doesn't work right, we should open an issue to revisit.

File: code/@ColorModel/ColorModel.m (hunk 1 above)
Comment (jedsinger) [reply code/@ColorModel/ColorModel.m]: I changed it because having "not yet set" as axis labels didn't look great for the demos at the PI meeting. "Fluorescence" was a nicer-looking fallback.

Comment (jakebeal) [reply code/@ColorModel/ColorModel.m]: OK, I'd like to keep it as 'not yet set' then. The axis labels _should_ always be reading either 'MEFL' or 'arbitrary units', and if they aren't, that's an issue that we should fix rather than patching around.

**Comment by jakebeal** [comment]
I think we also might run into issues about the placement of CSVs on TACC, and if we do, please open an issue and we'll solve it in a generalizable way.

**Comment by jedsinger** [comment]
Changed back from "Fluorescence" to "not yet set". Should be ready to merge.
